pkg/mappings/resources/events.go

```diff
@@ -49,6 +50,11 @@ func (s *eventMapper) HostToVirtual(ctx *synccontext.SyncContext, req types.Name
 		err = IgnoreAcceptableErrors(err)
 		if err != nil {
 			klog.Infof("Error retrieving involved object for %s/%s: %v", req.Namespace, req.Name, err)
+		} else if pObj.GetAnnotations()[constants.SyncResourceAnnotation] == "true" {
+			return types.NamespacedName{
+				Namespace: "default",
```

Comment:
maybe move this into a variable or a constant? 

Just a question, why are we hardcoding it to the default ns?

Reply:
> maybe move this into a variable or a constant?

Ok, will do.
> Just a question, why are we hardcoding it to the default ns?

If the event is not related to any specific synced resource then we don't have anything to go by to make the ns decision. Thus "default" seemed like a reasonable fallback. I've also considered adding an annotation for the virtual namespace, but default is enough for the initial use case.

Reply:
I see, that makes sense.